### ISSUE TITLE
An Empirical Study of Incorporating Pseudo Data into Grammatical Error Correction

### ISSUE THREAD

**Issue by fhiyo** [fhiyo/ml_papers#15]
## 0. 論文
https://arxiv.org/abs/1909.00502

## 1. どんなもの？

文法誤り訂正のタスクにおいて、
(1) 疑似データの生成方法, (2) seed corpusに何を選択するか, (3) 疑似データの比率やモデルの訓練方法などの設定 …①
を条件を変えて実験することで、疑似データの組み込みに対する理解を深めるもの。

実験から得られた最適な設定を用いたモデルがCoNLL-2014 test setとBEA-2019 shared taskにおいてSOTAを達成した。

## 2. 先行研究と比べてどこがすごい？

疑似データを取り込む際、①の条件は必ず決めなければならない事柄だがGECの分野ではそれらの条件に関するコンセンサスが取れていなかった。そこをどの条件で性能がどう変化するかを明らかにした点。

## 3. 技術や手法のキモはどこ？

誤った文法の文を生成するモデルで作成した大量の疑似データ (70M) でpre-trainし、BEA-trainでfine-tuneしたモデル(PRETLARGE)の作成。
これにSSE, R2Lのテクニックを組み合わせることでSOTAを達成

## 5. 議論はある？

4.4 (a) Joint Training or pretraining
で、疑似データの量を増やしたときにpretrainはF0.5が改善したがjoinは変化しなかった、と述べられている。これは疑似データが真のデータよりも量がかなり多くなると、jointの場合は疑似データの教師信号が支配的になってしまうが、pretrainの場合はfine-tuning時に真のデータを学習することができるためその問題が軽減されるから、と述べている (この説明はintuitiveであるとも言っている)。
